pkg/plugin/test-keeper/metrics.go

```diff
@@ -0,0 +1,72 @@
+package testkeeper
+
+import (
+	"github.com/arquillian/ike-prow-plugins/pkg/log"
+	gogh "github.com/google/go-github/github"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/arquillian/ike-prow-plugins/pkg/server"
+)
+
+const (
+	// WithTests is a label used in prometheus metrics pull_requests_total for pull request containing some tests.
+	WithTests = "with_tests"
+	// WithoutTests is a label used in prometheus metrics pull_requests_total for pull request without any tests.
+	WithoutTests = "without_tests"
+)
+
+var (
+	pullRequestsCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "pull_requests_total",
+		Help: "Total number of pull requests.",
+	}, []string{"full_name", "type"})
+	okWithoutTestsPullRequest = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "pull_requests_changed_file_size",
```

Comment:
`pull_requests_changed_file_count` or `pull_requests_size` 
WDYT?

Reply:
I was thinking about the same, but it's not size in `bytes` or some units. so it's quite difficult to understand while reading it.

Reply:
I think that it is clear in the context of PR that size refers to the number of changed files. Personally, I would vote for `pull_requests_size`